pkg/security/agent/client.go

```diff
@@ -205,7 +205,7 @@ func NewRuntimeSecurityClient() (*RuntimeSecurityClient, error) {
 				BaseDelay: time.Second,
 				MaxDelay:  time.Second,
 			},
-		}))
+		})) //nolint:staticcheck // TODO (ASC) fix grpc.Dial is deprecated
```

Comment:
Do we need 2 of these?

Reply:
Nope, this was a mistake, thanks for catching.